Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBProject.cs

```diff
@@ -13,39 +15,47 @@ public VBProject(VB.VBProject target, bool rewrapping = false)
         {
         }
 
-        public IApplication Application => throw new NotImplementedException();
-
-        public IApplication Parent => throw new NotImplementedException();
+        public IApplication Application => new Application(null);
+		
+        public IApplication Parent => new Application(null);
 
         public string ProjectId => HelpFile;
 
         public string HelpFile
         {
             get => IsWrappingNullReference ? string.Empty : Target.HelpFile;
-            set => Target.HelpFile = value;
+            set { if (!IsWrappingNullReference) Target.HelpFile = value; }
         }
 
         public string Description 
         {
             get => IsWrappingNullReference ? string.Empty : Target.Description;
-            set => Target.Description = value;
+            set { if (!IsWrappingNullReference) Target.Description = value; } 
         }
 
         public string Name
         {
             get => IsWrappingNullReference ? string.Empty : Target.Name;
-            set => Target.Name = value;
+            set { if (!IsWrappingNullReference) Target.Name = value; }
         }
 
-        public EnvironmentMode Mode => throw new NotImplementedException();
+        public EnvironmentMode Mode
+        {
+            get
+            {
+                if (IsWrappingNullReference) {return 0; }
```

Comment:
As a general rule, we prefer to have the scope on separate lines so that the indentation matches the scope.